src/Text/Pandoc/Definition.hs

```diff
@@ -301,6 +301,9 @@ data Block
     -- column alignments and widths (required), table head, table
     -- bodies, and table foot
     | Table Attr Caption [ColSpec] TableHead [TableBody] TableFoot
+    -- | Figure, with attributes, caption and caption position, width
+    -- (optional), and content (list of blocks)
+    | Figure Attr Caption [Block]
```

Comment:
are `caption position and width (optional)` encoded in the `Attr` somehow?

Reply:
I think this comment may be obsolete; an earlier version did represent caption position.